jenkins-pipelines/rolling-upgrade-alternator.jenkinsfile

```diff
@@ -5,7 +5,7 @@ def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
 
 rollingUpgradePipeline(
     backend: 'gce',
-    base_versions: ['4.4'],
+    base_versions: ['2020.1', '4.3', '2021.1'],
```

Comment:
The base_version change is branch special.

We don't support migrate from 2020.1 to master, and 2021.1 to master, so they can't be added to base_versions of master.

If we use SCT master to test Enterprise (enterprise branch), it's fine. And SCT pipeline will filter out the unsupported base_versions.

Reply:
Sorry, my bad, changed the branch (I wanted to merge it into branch-2021.1)